core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -89,6 +89,9 @@
       "Filename Offset Zero Pad Width";
 
   public static final String SCHEMA_CACHE_SIZE_CONFIG = "schema.cache.size";
```

Comment:
We should remove it. Deprecation doesn't seem required because it can't be used in a meaningful way now.